cmd/error.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type Class int
+
+//go:generate enumer -type Class -trimprefix Class -transform=lower -text
+const (
+	_ Class = iota
+	ClassMeter
+	ClassCharger
+	ClassVehicle
+	ClassTariff
+	ClassCircuit
```

Comment:
Error names don't match api/publishing names (circuits vs circuit, messaging vs. messager). Since mapping in the UI is done manually, this is not a problem, just wanted to make sure the singular is intentional.

Reply:
Das wird bei Fahrzeugen ähnlich sein. To be decided…

Reply:
Ja, ich würd sagen wir lassen das so. Bei Einzelgeräten sollten wir dann auch später den Identifier (db:23) mit rausgeben. Dann können wir per-device highlighten. Jetzt ists erstmal breit. Aber passt.

Reply:
Das ist schon so 😜